lib/lotus/mailer/configuration.rb

```diff
@@ -64,21 +161,155 @@ def root(value = nil)
         end
       end
 
+      # Prepare the mailers.
+      #
+      # The given block will be yielded when `Lotus::Mailer` will be included by
+      # a mailer.
+      #
+      # This method can be called multiple times.
+      #
+      # @param blk [Proc] the code block
+      #
+      # @return [void]
+      #
+      # @raise [ArgumentError] if called without passing a block
+      #
+      # @since 0.1.0
+      #
+      # @see Lotus::Mailer.configure
+      # @see Lotus::Mailer.duplicate
+      #
+      # @example Including shared utilities
+      #   require 'lotus/mailer'
+      #
+      #   module UrlHelpers
+      #     def comments_path
+      #       '/'
+      #     end
+      #   end
+      #
+      #   Lotus::Mailer.configure do
+      #     prepare do
+      #       include UrlHelpers
+      #     end
+      #   end
+      #
+      #   Lotus::Mailer.load!
+      #
+      #   module Comments
+      #     class New
+      #       # The following include will cause UrlHelpers to be included too.
+      #       # This makes `comments_path` available in the mailer context
+      #       include Lotus::Mailer
+      #
+      #       def form
```

Comment:
this helper confuses me because I don't think having HTML form in email is valid